rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketDefinitionController.java

```diff
@@ -104,20 +117,21 @@
   }
 
   /**
-   * This method imports a <b>list of {@link WorkbasketDefinitionRepresentationModel}</b>. This does
-   * not exactly match the REST norm, but we want to have an option to import all settings at once.
-   * When a logical equal (key and domain are equal) workbasket already exists an update will be
-   * executed. Otherwise a new workbasket will be created.
+   * This endpoint imports a list of Workbasket Definitions.
+   *
+   * <p>This does not exactly match the REST norm, but we want to have an option to import all
+   * settings at once. When a logical equal (key and domain are equal) Workbasket already exists an
+   * update will be executed. Otherwise a new Workbasket will be created.
    *
-   * @param file the list of workbasket definitions which will be imported to the current system.
-   * @return Return answer is determined by the status code: 200 - all good 400 - list state error
-   *     (referring to non existing id's) 401 - not authorized
+   * @title Import Workbaskets
+   * @param file the list of Workbasket Definitions which will be imported to the current system.
+   * @return no content
    * @throws IOException if multipart file cannot be parsed.
```

Comment:
what is a mulitpart file?

Reply:
That's the mime type for the file upload.

Reply:
See https://tools.ietf.org/html/rfc1867 (2.  HTML forms with file submission)